### PR TITLE
Added livewire support.

### PR DESCRIPTION
**Pull request by shankhadevpadam**
| Question                | Answer
| ----------------------- | -----------------------
| Issue or Enhancement    | Issue|Enhancement
| License                 | MIT

#### What's in this PR?
Added livewire support for package.

#### Checklist

- [x] I tested these changes.
- [ ] I have linked the related issues.

### CODE REVIEW

File: resources/views/master.blade.php [hunk 1]
```diff
@@ -83,6 +92,15 @@
         <script src="{{ mix(config('adminlte.laravel_mix_js_path', 'js/app.js')) }}"></script>
     @endif
 
+@if(config('adminlte.livewire'))
```
Comment (Shidersz) [inline resources/views/master.blade.php:95]: Can you fix tabulation?

**Comment by Shidersz** [comment]
It will be nice to document this new configuration option on the `README`, with at least a reference to [Livewire Site](https://laravel-livewire.com/) and the [Quickstart Documentation](https://laravel-livewire.com/docs/2.x/quickstart), since it requires installation of a new package to work.

File: resources/views/master.blade.php (hunk 1 above)
Comment (shankhadevpadam) [reply resources/views/master.blade.php]: Done

**Review by Shidersz — APPROVED** [review]

**Comment by lacodimizer** [comment]
@shankhadevpadam can you please add a note / docu to the readme?

**Comment by shankhadevpadam** [comment]
Sure

**Comment by Shidersz** [comment]
Please, note you may need to [rebase](https://www.atlassian.com/git/tutorials/rewriting-history/git-rebase) your branch, because the documentation `README.md` was recently updated. It will be also helpful to include why this support is needed. I have read the new Laravel 8 recommend the use of a new UI scaffolding named [Jetstream](https://jetstream.laravel.com/1.x/introduction.html) and `livewire` is related to it, but still has no time to experiment with this.

**Comment by Shidersz** [comment]
Sorry guys, but still can't see why we need to support [Laravel-Livewire](https://laravel-livewire.com/), it is based on [Tailwind CSS Framework](https://tailwindcss.com/), and for what I have read [here](https://laracasts.com/discuss/channels/elixir/can-one-use-both-tailwind-and-bootstrap-4-at-the-same-time) it is not so compatible with `Bootstrap 4` (but you can use both if you want). I believe this package should stimulate the usage of AdminLTE witch is based on Bootstrap 4...

**Comment by shankhadevpadam** [comment]
No, livewire work with any css framework. It not depends on tailwind css. Please read the docs. It is easy to build components without using any javascript framework. Livewire only use blade template.

**Comment by Shidersz** [comment]
@shankhadevpadam Ok, I'm going to give a deeper read when I found some time. Maybe the footer of the Livewire sites confuses me, it says:

```© Livewire 2020. Built with Laravel and Tailwind CSS.```

For the moment, I have made some improvements on the documentation, adding some links. When you have a moment, consider adding a basic example of what you can do with this package, it could be helpful for other readers.

**Comment by shankhadevpadam** [comment]
Built in tailwind in footer means, livewire not build in tailwind. It tells, that site was built in tailwind. 🙂🙂🙂

**Pull request merged**